pkgs/nix-tools/darwin-rebuild.sh

```diff
@@ -187,7 +187,7 @@ fi
 
 if [ "$action" = list ] || [ "$action" = rollback ]; then
   if [ "$USER" != root ] && [ ! -w $(dirname "$profile") ]; then
-    sudo -H nix-env -p "$profile" "${extraProfileFlags[@]}"
+    sudo -H --preserve-env=PATH env nix-env -p "$profile" "${extraProfileFlags[@]}"
```

Comment:
Is the `env` command necessary if you’re already using the `--preserve-env=PATH` flag?

Reply:
Hey @Enzime , thanks for taking a look. Exactly, `--preserve-env=PATH` will preserve the ENV variable in the scope of executing the `sudo` command, but it's not available when `sudo` is actually looking for the binary to execute. That's the reason we use `env` so it _evaluates_ this ENV variable first and searches for the binary in the PATH. 
Does this make sense?

Reply:
Yeah that makes sense :+1: